### ISSUE TITLE
Question: Is it possible to connect with remote driver to my self-hosted selenium grid?

### ISSUE THREAD

**Issue by UmutAlihan** [ArchiveBox/ArchiveBox#979]
Hi,

I am hosting a selenium grid deployment on my home server. My other projects connect to the grid via remote driver and execute browser actions there. I am kind of collecting all browser actions in single place for all my projects.

Can I also use my grid with remote driver on ArchiveBox? Is it possible, should I request it as a new feature?

Any help is appreciated.

**Comment by pirate** [comment]
Nope, this is not supported and not planning on it anytime in the near/medium future. you can always run ArchiveBox itself on a remote server and send URLs to it with the API or have it poll for URLs with `archivebox schedule`.

**Comment by pirate** [comment]
Going to close this for now, as we may add our own internal ability to use remote ArchiveBox instances to archive pages: https://github.com/ArchiveBox/ArchiveBox/issues/786
